timesketch/views/spa.py

```diff
@@ -16,13 +16,28 @@
 from __future__ import unicode_literals
 
 from flask import Blueprint
+from flask import redirect
 from flask import render_template
 from flask_login import login_required
 
 # Register flask blueprint
 spa_views = Blueprint('spa_views', __name__)
 
 
+@spa_views.route(
+    '/sketch/<int:sketch_id>/explore/view/<int:view_id>/', methods=['GET'])
+@login_required
+# pylint: disable=unused-argument
+def redirect_view(sketch_id, view_id):
+    """Generates the template.
```

Comment:
this feels a bit weird....

the function is called `redirect_view` but the docstring says `generate the template` and then all it does is return a `redirect` call.. so the function name and content seem to match but the docstring sounds a bit weird